server/src/services/album.service.ts

```diff
@@ -106,12 +108,18 @@ export class AlbumService extends BaseService {
     });
     const assetIds = [...allowedAssetIdsSet].map((id) => id);
 
+    const metadata = await this.userRepository.getMetadata(auth.user.id);
+    const preferences = mapPreferences(getPreferences(metadata));
+
+    const assetOrder = preferences.albums.defaultAssetOrder ?? AssetOrder.DESC;
```

Comment:
I would just merge this into line 122 and I don't think it can be undefined once the map call is removed.

Reply:
Done.